crates/flox/doc/flox-run.md

```diff
@@ -11,21 +11,58 @@ flox-run - run app from current project
 
 # SYNOPSIS
 
-flox [ `<general-options>` ] run [ `<options>` ] [ -- [ `<command args>` ... ] ]
+flox [ `<general-options>` ] run [ `<run-options>` ] [ -- [ `<command args>` ... ] ]
 
 # DESCRIPTION
 
 Run flake application from the requested package (or "installable").
 If not provided `flox` will prompt for you to select from the list of known packages.
+`flox run` uses `nix run` under the hood to execute the so-called installable. Which makes it possible to run 
+packages from any Nix flake that exposes the `app` attribute in its outputs.
 
-# OPTIONS
+# EXAMPLES
 
-```{.include}
-./include/general-options.md
-./include/development-options.md
+## Running applications in the current working directory
+
+If `flox run` is called without any arguments, it will ask the user which application they want to use.
+
+```console
+$ flox run
+? Select a packageapp for flox run  
```

Comment:
Huh is `packageapp` a typo we need to fix somewhere else?

Reply:
I believe it's coming from here.

https://github.com/flox/flox/blob/8bd2556141ccd8fd81bf69b755a5a8e160577616/crates/flox/src/utils/mod.rs#LL381C3-L381C3

I executed that command in the [floxdocs repo](https://www.github.com/flox/floxdocs).

Reply:
`packageapp` does not seem like something we should be displaying to users 🙈 

Not relevant to this PR though

Reply:
should be interspersed with 'or', it's a formatting bug